tests/unit/server/bundle_manager_test.py

```diff
@@ -149,6 +153,23 @@ def get_sample_workers_list(self):
         ]
         return workers_list
 
+    def test_init_should_parse_config(self):
+        self.codalab_manager.config = {
+            'workers': {
+                'default_cpu_image': 'codalab/default-cpu:latest',
+                'default_gpu_image': 'codalab/default-gpu:latest',
+                'max_request_time': '50s',
+                'max_request_memory': '100m',
+                'min_request_memory': '10m',
+                'max_request_disk': '10m',
+            }
+        }
+        self.bundle_manager = BundleManager(self.codalab_manager)
+        self.assertEqual(self.bundle_manager._max_request_time, 50)
+        self.assertEqual(self.bundle_manager._max_request_memory, 100 * 1024 * 1024)
+        self.assertEqual(self.bundle_manager._min_request_memory, 10 * 1024 * 1024)
+        self.assertEqual(self.bundle_manager._max_request_disk, 10 * 1024 * 1024)
```

Comment:
Don't we already have this above?

Reply:
I just moved the old bundle manager tests we already had previously to this file -- don't want to refactor that, beyond the scope of this PR. Didn't want to remove them because we might as well keep them.